dist/angular-validation.js

```diff
@@ -619,23 +620,23 @@
 
                         })();
 
-                        $timeout(function() {
```

Comment:
@prugel 

I prefer noValidationMessage should be set here.

I don't want to check it every time within `validFunc` and `invalidFunc`

Because using `noValidationMessage` is the very specific/special case only not for the general.